pkgs/top-level/all-packages.nix

```diff
@@ -6211,6 +6210,8 @@ in
 
   mpack = callPackage ../tools/networking/mpack { };
 
+  mpi = openmpi;
```

Comment:
Please add that either above openmpi or to aliases.

Reply:
aliases is for renames that are deprecated. This is a form of configuration.

Reply:
I moved it next to `openmpi`.